cirq/contrib/quimb/Cirq-to-Tensor-Networks.ipynb

```diff
@@ -1,5 +1,23 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import importlib.util\n",
+    "\n",
+    "if importlib.util.find_spec('cirq') is None: \n",
+    "    print(\"installing cirq\")\n",
+    "    !pip install --quiet cirq\n",
```

Comment:
Optional: Running the shell command comes with a slight risk that pip will install the package into a location different from the one used by the colab kernel. There is a special colab magic `%pip` that ensures installation into the appropriate location. Not sure when it was added though.

Reply:
Huh, let me look into this, thanks @viathor!